common-lib/src/cmr/common/config.clj

```diff
@@ -233,6 +233,20 @@
              :serviceoption ["1.0.0"]}
    :parser #(json/parse-string % true)})
 
+(defconfig approved-pipeline-documentation
+  "This is the feature toggle for activating documentation for
```

Comment:
If the documentation is to follow the actual approved generic documents do we need this toggle?  Can we just use the approved-pipeline-documents defconfig?

Reply:
We can and that's how it was functioning before, this is for the case if we want the generic fully functioning but, we don't want to have documentation appear for it, perhaps that is not a valid use case?

Reply:
I don't think it is a valid use case.

Reply:
Altered to keep the function but, have it set to the default for the other generic defconfig